Algorithmia/Data/AlgoDataObject.swift

```diff
@@ -87,6 +87,14 @@ class AlgoDataObject {
         return self.path.substring(from: range.upperBound)
     }
     
+    
+    /// Return full path of File/Directory. e.g., data://.my/foo/bar.txt
+    ///
+    /// - returns: full path of File/Directory
+    func fullPath() -> String {
+        return "data://"+self.path
```

Comment:
We support other data sources like s3 and dropbox. The `path` might look like `dropbox://myfile` in which case you wouldn't want to prefix it with `data://`.  So I think you should add `data://` if the `path` doesn't already contain `://`. Fwiw, I wanted to compare with other clients, and [we have this same problem in the Java client](https://github.com/algorithmiaio/algorithmia-java/issues/10).

Also, I prefer a name like `toDataURI` instead of `fullPath` because we call this a Data URI in all of our docs.